integreat_cms/static/src/js/translations/budget-graph.ts

```diff
@@ -0,0 +1,44 @@
+import { Chart, ChartConfiguration, DoughnutController, ArcElement } from "chart.js";
+
+Chart.register(DoughnutController, ArcElement);
+
+window.addEventListener("load", async () => {
+    const graph = document.querySelector("#budget-graph") as HTMLElement;
+    if (!graph) {
+        return;
+    }
+
+    const budgetUsed = parseInt(graph.dataset.budgetUsed, 10);
+    const budgetTotal = parseInt(graph.dataset.budgetTotal, 10);
+
+    const getGraphColor = () => {
+        // hue of 0 is red, hue of 128 is green
+        const maxHueValue = 128;
+        const fractionUsed = budgetUsed / budgetTotal;
+
+        // Return a Hue-Saturation-Luminance string where Hue depends on the used budget
+        return `hsl(${(1 - fractionUsed) * maxHueValue}, 100%, 50%)`;
+    };
+
+    const data = {
+        datasets: [
+            {
+                data: [budgetUsed, budgetTotal - budgetUsed],
+                backgroundColor: [getGraphColor(), "lightgray"],
+                circumference: 180,
+                rotation: -90,
+            },
+        ],
+    };
+
+    const _: Chart = new Chart("budget-graph", {
+        type: "doughnut",
```

Comment:
I know there has been some discussion lately if it would be better to use bar charts instead of doughnut, since they take up less height. @osmers Do we want to stick to the design with the doughnut or do we want to change it to a bar chart (as for text understandibility).

Reply:
In the case of this specific chart, we have lots of space, since there's not really anything else going on in this column of the management page. I'd rather we changed both this one and the HIX widget at the same time, should that be decided, and leave this one as a doughnut for now.

Though I'm of course open to changing it to a bar if that's desired.

Reply:
+1 for keeping it a doughnut for the time being

Reply:
Alright :)

Reply:
I'm ok with changing both at the same time :) should I open an issue for both?